common/web/types/src/consts/virtual-key-constants.ts

```diff
@@ -174,13 +187,13 @@ export const ISOVirtualKeyMap: KeyMap = [
 
 export const JISVirtualKeyMap: KeyMap = [
   // [Hankaku/Zenkaku] 1 2 3 4 5 6 7 8 9 0 - ^ ¥ [bksp]
-  [ k.K_1, k.K_2, k.K_3, k.K_4, k.K_5, k.K_6, k.K_7, k.K_8, k.K_9, k.K_0, k.K_HYPHEN, k.K_EQUAL, k.K_BKSLASH /* YEN */ ],
+  [ k.K_BKQUOTE, k.K_1, k.K_2, k.K_3, k.K_4, k.K_5, k.K_6, k.K_7, k.K_8, k.K_9, k.K_0, k.K_HYPHEN, k.K_EQUAL, k.K_BKSLASH /* YEN */ ],
   // [tab] Q W E R T Y U I O P @«`» [ [enter]
   [ k.K_Q, k.K_W, k.K_E, k.K_R, k.K_T, k.K_Y, k.K_U, k.K_I, k.K_O, k.K_P, k.K_BKQUOTE, k.K_LBRKT  ],
   // [caps] A S D F G H J K L ; : ] [enter]
   [ k.K_A, k.K_S, k.K_D, k.K_F, k.K_G, k.K_H, k.K_J, k.K_K, k.K_L, k.K_COLON, k.K_QUOTE, k.K_RBRKT ],
   // [shift] Z X C V B N M , . / _ [shift]
-  [ k.K_Z, k.K_X, k.K_C, k.K_V, k.K_B, k.K_N, k.K_M, k.K_COMMA, k.K_PERIOD, k.K_SLASH, k.K_oE2 /* ろ */ ],
+  [ k.K_oE2, k.K_Z, k.K_X, k.K_C, k.K_V, k.K_B, k.K_N, k.K_M, k.K_COMMA, k.K_PERIOD, k.K_SLASH, k.k_oC1 /* ろ */ ],
```

Comment:
My JIS keyboard doesn't have a key between Shift and Z:

![20230925_091137](https://github.com/keymanapp/keyman/assets/4498365/1e386f1b-9228-4356-8516-9c9dda020deb)

FWIW, in Windows, the virtual key code at far-right (scan code 0x73) is `K_oE2` when using Japanese base keyboard, or `K_oC1` when using US base keyboard. Yay VK!

We should probably use `K_oC1` internally for scan code 0x73 in Keyman Engine for Windows, which we need to track as a separate issue. See also the KS discussion above.

Reply:
OK, mine didn't have one there either. Looks like a copy and paste. thanks

Reply:
This is broken in the CLDR file, so opened a linked PR there.